<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>670</width>
    <height>538</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(20,20,20);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="enterButton">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>490</y>
      <width>31</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	border-radius: 6px;
	background-color: rgb(30,30,30);
	color: rgb(155,155,155);
	
	font: 10pt &quot;Cambria&quot;;
}
QPushButton:pressed{
	border-radius: 6px;
	background-color: rgb(90,90,90);
	color: rgb(220,220,220);
	
	font: 10pt &quot;Cambria&quot;;
}</string>
    </property>
    <property name="text">
     <string>-&gt;</string>
    </property>
   </widget>
   <widget class="QListWidget" name="msgList">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>60</y>
      <width>371</width>
      <height>341</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QListWidget{
	border-radius: 16px;
	background-color: rgb(30,30,30);
	color: rgb(155,155,155);
	font: 12pt &quot;Cambria&quot;;
	padding:12px;
	
}

QListWidget::item {
    background-color: rgb(40,40,40);
	border:6px solid rgb(30,30,30);
	border-radius: 16px;
}
</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>490</y>
      <width>101</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	border-radius: 6px;
	background-color: rgb(30,30,30);
	color: rgb(155,155,155);
	
	font: 10pt &quot;Cambria&quot;;
}
QPushButton:pressed{
	border-radius: 6px;
	background-color: rgb(90,90,90);
	color: rgb(220,220,220);
	
	font: 10pt &quot;Cambria&quot;;
}</string>
    </property>
    <property name="text">
     <string>Добавить файл</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="msgEdit">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>410</y>
      <width>371</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QTextEdit{
background-color: rgb(40, 40, 40);
color: rgb(155, 155, 155);
font: italic 10pt &quot;Cambria&quot;;
border-radius: 12px;
border-style: solid;
border-color: rgb(0, 0, 0)
}
</string>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Cambria'; font-size:10pt; font-weight:400; font-style:italic;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="placeholderText">
     <string>Введите сообщение</string>
    </property>
   </widget>
   <widget class="QListWidget" name="userList">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>271</width>
      <height>451</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QListWidget{
	border-radius: 16px;
	background-color: rgb(30,30,30);
	color: rgb(155,155,155);
	font: 24pt &quot;Cambria&quot;;
	padding:8px;
	text-align: center;
}

QListWidget::item {
    background-color: rgb(40,40,40);
	border:1px solid  #323232;
	border-radius: 16px;

}

QListWidget::item:selected {
    background-color: rgb(0,0, 0);
	border:1px solid  #323232;
	border-radius: 16px;
	color: rgb(155,155,155);
	font: 24pt &quot;Cambria&quot;;
}

QListWidget::item:hover {
    background-color: rgb(10, 10, 10)
	color: rgb(155,155,155);
	font: 24pt &quot;Cambria&quot;;
	border:1px solid  #323232;
	border-radius: 16px;
}</string>
    </property>
   </widget>
   <widget class="QLabel" name="UserName">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>10</y>
      <width>231</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(155,155,155);
font: 20pt &quot;Cambria&quot;;
background-color: rgb(30,30,30);
border-radius: 10px;
</string>
    </property>
    <property name="text">
     <string>UserNameHere</string>
    </property>
    <property name="textFormat">
     <enum>Qt::AutoText</enum>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>10</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(175,175,175);
font: 75 italic 18pt &quot;Cambria&quot;;
</string>
    </property>
    <property name="text">
     <string>SomeMessenger</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
